app/server/lib/DocApi.ts

```diff
@@ -170,6 +170,7 @@ export class DocWorkerApi {
     const canView = expressWrap(this._assertAccess.bind(this, 'viewers', false));
     // check document exists (not soft deleted) and user can edit it
     const canEdit = expressWrap(this._assertAccess.bind(this, 'editors', false));
+    const canCreate = expressWrap(this._checkAnonymousCreation.bind(this));
```

Comment:
The name of this piece of middleware could be misleading, when seen side by side with `canView` and `canEdit`. Something like `isAnonCreationEnabled` might be more clearly distinct, or `checkAnonymousCreation`.

Reply:
Done